photon-core/src/main/java/org/photonvision/vision/processes/PipelineManager.java

```diff
@@ -70,10 +71,19 @@ public class PipelineManager {
         if (userPipelines.isEmpty()) addPipeline(PipelineType.Reflective);
 
         calibration3dPipeline = new Calibrate3dPipeline(uniqueName);
+
+        // We know that at this stage, VisionRunner hasn't yet started so we're good to do this from
```

Comment:
Will this always be true?

Reply:
Yes absolutely true. PipelineManager is a prerequisite for VisionRunner existing